lib/test_queue/runner.rb

```diff
@@ -337,6 +349,7 @@ def distribute_queue
               # If we have a slave from a different test run, don't respond, and it will consider the test run done.
               sock.write("OK\n")
               remote_workers += num
+              quorum_members += num
```

Comment:
Hmm. That would only happen if we have run out of tests to run, and we could guard on that as well.

My concern with the remote worker count is something connecting 10 times quickly and failing. We'll get 10 workers but the intent will be lost. We haven't seen exactly that failure mode, though, so perhaps I'm optimizing prematurely.